server/sync/leaderboard_scheduler.go

```diff
@@ -0,0 +1,198 @@
+package sync
+
+import (
+	"strings"
+	"time"
+
+	"github.com/samouraiworld/topofgnomes/server/models"
+)
+
+// StartPersonalLeaderboardScheduler starts a loop that periodically checks
+// active LeaderboardConfig entries and posts the leaderboard accordingly.
+func (s *Syncer) StartPersonalLeaderboardScheduler() {
+	go func() {
+		ticker := time.NewTicker(1 * time.Minute)
+		defer ticker.Stop()
+		for range ticker.C {
+			s.runDueLeaderboardJobs()
+		}
+	}()
+}
+
+func (s *Syncer) runDueLeaderboardJobs() {
+	var configs []models.LeaderboardConfig
+	now := time.Now().UTC()
+	if err := s.db.Where("active = ? AND (next_run_at IS NULL OR next_run_at <= ?)", true, now).Find(&configs).Error; err != nil {
```

Comment:
no need to pass true as argument for booleans
```suggestion
	if err := s.db.Where("active AND (next_run_at IS NULL OR next_run_at <= ?)", now).Find(&configs).Error; err != nil {
```